compiler/rustc_typeck/src/check/wfcheck.rs

```diff
@@ -798,26 +800,26 @@ fn check_where_clauses<'tcx, 'fcx>(
                 params: FxHashSet<u32>,
             }
             impl<'tcx> ty::fold::TypeVisitor<'tcx> for CountParams {
```

Comment:
That folder seems weird to me, why are we storing the params in a hashmap if we only care about if there is more than one of them?

Using an option is probably more appropriate here but whatever :shrug: Isn't really relevant for this PR